CHANGELOG.md

```diff
@@ -0,0 +1,51 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+
+## [unreleased](https://github.com/joeloskarsson/neural-lam/compare/v0.1.0...HEAD)
+
+### Added
+
+- new metrics (`nll` and `crps_gauss`) and `metrics` submodule, stddiv output option
+  [c14b6b4](https://github.com/joeloskarsson/neural-lam/commit/c14b6b4323e6b56f1f18632b6ca8b0d65c3ce36a)
+  @joeloskarsson
+
+- ability to "watch" metrics and log
+  [c14b6b4](https://github.com/joeloskarsson/neural-lam/commit/c14b6b4323e6b56f1f18632b6ca8b0d65c3ce36a)
+  @joeloskarsson
+
+- pre-commit setup for linting and formatting
+  [\#6](https://github.com/joeloskarsson/neural-lam/pull/6), [\#8](https://github.com/joeloskarsson/neural-lam/pull/8)
+  @sadamov, @joeloskarsson
+
+### Changed
+
+- moved batch-static features ("water cover") into forcing component return by `WeatherDataset`
+  [\#13](https://github.com/joeloskarsson/neural-lam/pull/13)
+  @joeloskarsson
+
+- change validation metric from `mae` to `rmse`
+  [c14b6b4](https://github.com/joeloskarsson/neural-lam/commit/c14b6b4323e6b56f1f18632b6ca8b0d65c3ce36a)
+  @joeloskarsson
+
+- compute `rmse` after spatial averaging
```

Comment:
The change is to take sqrt after averaging all the forecasts. It was always after spatial averaging. See e.g. difference between v1 and v2 of WB2 https://arxiv.org/abs/2308.15560.

Reply:
I don't quite understand what you mean by "averaging all the forecasts". Is there a figure in the WeatherBench2 article you're referring? Or is it something discussed in v2 that was missing in v1 of that article?

Reply:
Sorry, I realize that was not very clear. I was referencing the RMSE definition in the WB2 paper (eq.2 in both versions). It is somewhat different in v1 and v2. A good description for this change would be "change RMSE definition to compute sqrt after all averaging".

Reply:
thank you :)